asoh/estimators/online/__init__.py

```diff
@@ -1 +1,136 @@
-""" Collection of online estimators """
+"""
+Collection of online estimators
+
+Here, we include base classes to facilitate  definition of online estimators and hidden states, as well as to establish
+interfaces between online estimators and models, transient states, and A-SOH parameters.
+"""
+from abc import abstractmethod
+from typing import Union, List
+
+import numpy as np
+from pydantic import BaseModel, Field
+
+
+class MultivariateRandomDistribution(BaseModel, arbitrary_types_allowed=True):
+    """
+    Base class to help represent a multivariate random variable
+    """
+
+    @property
+    def expected_value(self) -> np.ndarray:
+        """
+        Provides expected value of distribution
+        """
+        if hasattr(self, 'mean'):
```

Comment:
Should we just add a `mean` property to this class if it's expected. Something like 

```python

@property
@abstractmethod
def mean(self) -> np.ndarray:
    pass
```

Reply:
@WardLT I think that's a good idea! Going to push a new version soon.

Reply:
Actually, the reason I did not do that is that, for some other child classes, `mean` is a `pydantic` `Field`, and not a property, so we cannot force it to be an `abstractproperty` in my opinion. @WardLT I don't quite see a way around that...

Reply:
Defining `mean` to be a pydantic field in subclasses also won't be a problem, as it will still provide the same "property" interface. Unless I'm mistaken, let's go ahead and add the `mean` property.

Reply:
Fixed by enforcing `get_mean` abstractmethod